rules-reviewed/eap6/jsp/jsp.windup.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="eap6-jsp"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            JSP related rules for migration from different server to JBoss EAP 6+.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.3.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.3.0.Final" />
+        </dependencies>
+        <targetTechnology id="eap" versionRange="[6,)" />
+        <targetTechnology id="java-ee" versionRange="[6,)" />
+        <tag>reviewed-2016-04-25</tag>
+    </metadata>
+    <rules>
+        <rule id="jsp-01000">
+            <when>
+                <filecontent filename="{*}.jsp" pattern="import=&quot;&quot;" />
```

Comment:
It works in wildfly and I don't know any reason why it wouldn't be legal. Single-quotes and double-quotes are typically interchangeable in contexts like this.